torchdynamo/utils.py

```diff
@@ -712,12 +712,13 @@ def wrap_fake_exception(fn):
         try:
             return fn()
         except UnsupportedFakeTensorException as e:
-            from .exc import FakeTensorError
+            from .exc import unimplemented
 
-            raise FakeTensorError(
+            log.warning(
                 f"Unsupported: {e.reason} with fake tensor propagation. "
                 "Run with config.fake_tensor_propagation=False"
-            ) from e
+            )
+            raise unimplemented(f"Unsupported: {e.reason} with fake tensor propagation")
```

Comment:
this is more targeted for devs/export path, but maybe it should be more tailored to users

Reply:
As long as that config offers a path to avoiding the break, I'd leave the comment reference to it. Both for export, and because some graphs breaks can become perf hits.

Reply:
> because some graphs breaks can become perf hits

I'll note that for all of the unsupported fake tensor stuff we don't support in torchinductor either